index.js

```diff
@@ -28,6 +28,7 @@ const config = {
 		'space-in-parens': ['error', 'never'],
 		'arrow-spacing': 'error',
 		'arrow-parens': 'error',
+		'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
```

Comment:
Is this intentional? Maybe you meant to add this new rule as a separate pull request?

Reply:
Note to self: this is saying to ban `var foo = function ...` but allow `var foo = (...) => ...`

While I don't really disagree with this change, I'm pretty sure it will be a yak shave in some repos :)

Reply:
I don't remember why I made this change so I have removed it.